README.md

```diff
@@ -1,2 +1,23 @@
+ The NWB format and DANDI archive not only facilitate the exchange of datasets among researchers but also streamline data for software tool developers. The highly standardized nature of the NWB format together with the easy accessibility of the datasets in the DANDI archive make the platform an ideal sandbox for emerging software domains such as electrophysiological spike sorting, calcium imaging segmentation, and behavioral pose estimation.
+
+Despite the numerous advantages offered by NWB and DANDI, widespread adoption is hindered by a multitude of challenges, each presenting a distinct hurdle for laboratories seeking to seamlessly integrate these resources into their research workflows. First, adopting NWB requires at least a summary understanding of its data organization and principles, the sheer scale of which can be daunting for those unfamiliar with its structure. This learning curve poses a substantial obstacle, slowing down the assimilation of NWB standards into the daily practices of research labs. In addition, the utilization of data Application Programming Interfaces (APIs), such as _pyNWB_ or _MatNWB_, demands a level of programming expertise that may be beyond the reach of researchers with limited coding experience. This technical barrier creates an accessibility gap, restricting the adoption of NWB to those proficient in programming or placing an outsized burden on those, often junior, lab members with the necessary expertise.
+
+If labs do make the decision to adopt the NWB standard, they often find themselves motivated to convert their backlogged data to the NWB format. Recognizing the advantages of NWB, labs aim to ensure that their extensive historical datasets align with this standardized framework, enhancing data interoperability, collaboration, and long-term accessibility. However, this effort tends to be a substantial bottleneck in the transition to NWB. The accumulation of historical data, often stored in diverse and proprietary formats, poses a formidable challenge, as researchers must now navigate through and transform these datasets to comply with NWB. The sheer volume of legacy data, combined with its inherent heterogeneity, turns the conversion process into a labor-intensive endeavor. Further hampering adoption efforts is the dearth of individual incentives for researchers to prioritize this time-consuming task. In the fast-paced environment of scientific research, where immediate outcomes and results often take precedence, dedicating time and resources to the retrospective conversion of historical data may not be perceived as an urgent priority for individual researchers. The absence of tangible, career-advancing rewards or incentives further hampers the motivation to address this critical aspect of NWB adoption.
```

Comment:
We should mention how a growing number of institutions are actually leveraging human resources to help with the labor side of this endeavor

NYU: the librarians (I think of the scientific computing center?) offered to help some labs with the task
Brown: Ritt group, obviously [good opportunity to cite their paper]
Princeton: has software professionals that labs can request the time of for projects like this
U19: data core is specifically the go-to person to handle this for multiple labs
Non-profit institutions (Simons, ASAP): contracts with neurodata consulting companies such as CatalystNeuro

Will try to remember if we've heard of others

Should also give honorable mention to NWB export of for-profit data management and processing services like DataJoint, Inscopix, Allegro (NeuroNexus)